node/src/indexer.rs

```diff
@@ -120,7 +121,11 @@ async fn handle_block(
                             epsilon,
                             delta,
                             entropy,
+                            time_added: Instant::now(),
```

Comment:
nit: how about we keep it consistent with `timestamp` everywhere else since timestamp is usually associated to when something is created anyways